core/new-gui/src/app/workspace/service/execute-workflow/execute-workflow.service.ts

```diff
@@ -225,14 +225,14 @@ export class ExecuteWorkflowService {
     this.workflowWebsocketService.send("WorkflowKillRequest", {});
   }
 
-  public addExecutionInteraction(): void {
+  public takeGlobalCheckpoint(): void {
     if (
       this.currentState.state === ExecutionState.Uninitialized ||
       this.currentState.state === ExecutionState.Completed
     ) {
       throw new Error("cannot add interaction to workflow, the current execution state is " + this.currentState.state);
```

Comment:
error message is not related to checkpoint, it is still "interaction" related.

Reply:
changed